labs-ios-starter/Map/LocationSearchTableViewController.swift

```diff
@@ -47,38 +51,57 @@ class LocationSearchTableViewController: UITableViewController {
     // MARK: - Table view data source
 
     override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return matchingItems.count
+        return citySearchResults.count
     }
     
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let cell = tableView.dequeueReusableCell(withIdentifier: "cell")!
-        let selectedItem = matchingItems[indexPath.row].placemark
-        cell.textLabel?.text = selectedItem.name
-        cell.detailTextLabel?.text = parseAddress(selectedItem: selectedItem)
+        //let selectedItem = matchingItems[indexPath.row].placemark
+        //cell.textLabel?.text = selectedItem.name
```

Comment:
Before we turn this in, let's make sure all commented code is removed

Reply:
I think you are commenting on an "Outdated" earlier commit - I looked at [/files](https://github.com/Lambda-School-Labs/cityspire-c-ios/pull/4/files) and see that these are not included by the final commit